### PR TITLE
wasmer import refactor

### PR DESCRIPTION
**Pull request by andrei-marinica**
Decoupled import declaration from wasmer.

Import declaration belongs with the elrondapi part of the VM, while the wasmer imports are a specific executor implementation.

Done in preparation for wasmer 2.2.

### CODE REVIEW

File: arwen/elrondapi/manBufOps.go [hunk 1]
```diff
@@ -119,11 +119,6 @@ func ManagedBufferImports(imports *wasmer.Imports) (*wasmer.Imports, error) {
 		return nil, err
 	}
 
-	imports, err = imports.Append("mBufferSetBytes", v1_4_mBufferSetBytes, C.v1_4_mBufferSetBytes)
```
Comment (camilbancioiu) [inline arwen/elrondapi/manBufOps.go:122]: Why was `mBufferSetBytes()` removed?

Comment (andrei-marinica) [reply arwen/elrondapi/manBufOps.go:122]: Was duplicate! 
Discovered because I actually added a check for duplicates!

**Review by camilbancioiu — DISMISSED** [review]

File: arwen/elrondapi/manBufOps.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (andrei-marinica) [reply arwen/elrondapi/manBufOps.go:122]: It was duplicate. I discovered that because I added a check for duplicates.

**Review by sasurobert — APPROVED** [review]

File: arwen/host/arwen.go [hunk 2]
```diff
@@ -138,43 +137,12 @@ func NewArwenVM(
...
-	if err != nil {
-		return nil, err
-	}
-
-	imports, err = cryptoapi.CryptoImports(imports)
-	if err != nil {
-		return nil, err
-	}
+	imports := elrondapimeta.NewEIFunctions()
+	PopulateAllImports(imports)
```
Comment (camilbancioiu) [inline arwen/host/arwen.go:142]:
```suggestion
	err := PopulateAllImports(imports)
	if err != nil {
		return err
	}
```

**Review by camilbancioiu — DISMISSED** [review]

**Review by sasurobert — APPROVED** [review]

**Pull request merged**